### PR TITLE
e2e using new harmonize content

### PR DESCRIPTION
**Pull request by ayuwono**
This change is to use new custom harmonize content plugin for sku and SKU property. Also using xpath to get the price property

### CODE REVIEW

File: quick-start/e2e/qa-data/plugins/contentWithOptions.sjs [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
-      
-      // return the instance object
-      return {
-        '$attachments': attachments,
-        '$type': 'Product',
-        '$version': '0.0.1',
-        'sku': sku,
-        'title': title,
-        'price': price,
-        'opt1': value1,
```
Comment (aebadirad) [inline quick-start/e2e/qa-data/plugins/contentWithOptions.sjs]: are we still using opt1 and opt2 for tests anywhere else?

Comment (bsrikan) [reply quick-start/e2e/qa-data/plugins/contentWithOptions.sjs]: I do see opt1 in 1 traces test 'should verify the traces viewer page'
and 3 jobs test
'should verify the loaded data'
'should verify the harmonized data with sku as original property'
'should verify the harmonized data with SKU as original property'

Tests dont use opt2 anywhere.

Comment (bsrikan) [reply quick-start/e2e/qa-data/plugins/contentWithOptions.sjs]: if I am not wrong, I think opts was missed out and adding them to the updated custom content plugin should be good enough. Will confirm with @ayuwono

**Comment by ayuwono** [comment]
Good catch, modifying it right now

**Comment by ayuwono** [comment]
@aebadirad , fixed the content with options values

**Pull request merged**